2021-04-26 02:39:28,653 INFO    MainThread:47009 [wandb_setup.py:_flush():69] setting env: {}
2021-04-26 02:39:28,653 INFO    MainThread:47009 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-26 02:39:28,654 INFO    MainThread:47009 [wandb_init.py:_log_setup():336] Logging user logs to /home/raghavauppuluri/research/deep_learning/binary_classifer/wandb/run-20210426_023928-1is0y6tl/logs/debug.log
2021-04-26 02:39:28,654 INFO    MainThread:47009 [wandb_init.py:_log_setup():337] Logging internal logs to /home/raghavauppuluri/research/deep_learning/binary_classifer/wandb/run-20210426_023928-1is0y6tl/logs/debug-internal.log
2021-04-26 02:39:28,654 INFO    MainThread:47009 [wandb_init.py:init():369] calling init triggers
2021-04-26 02:39:28,654 INFO    MainThread:47009 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'epochs': 25, 'batch_size': 4, 'num_workers': 2, 'learning_rate': 0.001, 'optimizer': 'adam', 'step_size': 7, 'gamma': 0.1}
2021-04-26 02:39:28,654 INFO    MainThread:47009 [wandb_init.py:init():418] starting backend
2021-04-26 02:39:28,655 INFO    MainThread:47009 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-26 02:39:28,724 INFO    MainThread:47009 [backend.py:ensure_launched():123] starting backend process...
2021-04-26 02:39:28,731 INFO    MainThread:47009 [backend.py:ensure_launched():127] started backend process with pid: 47063
2021-04-26 02:39:28,732 INFO    MainThread:47009 [wandb_init.py:init():423] backend started and connected
2021-04-26 02:39:28,735 INFO    MainThread:47009 [wandb_init.py:init():465] updated telemetry
2021-04-26 02:39:28,735 INFO    MainThread:47009 [wandb_init.py:init():484] communicating current version
2021-04-26 02:39:29,359 INFO    MainThread:47009 [wandb_init.py:init():489] got version response 
2021-04-26 02:39:29,359 INFO    MainThread:47009 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-26 02:39:29,506 INFO    MainThread:47009 [wandb_init.py:init():522] starting run threads in backend
2021-04-26 02:39:29,582 INFO    MainThread:47009 [wandb_run.py:_console_start():1524] atexit reg
2021-04-26 02:39:29,583 INFO    MainThread:47009 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-26 02:39:29,584 INFO    MainThread:47009 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-26 02:39:29,728 INFO    MainThread:47009 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-26 02:39:29,728 INFO    MainThread:47009 [wandb_init.py:init():546] run started, returning control to user process
2021-04-26 02:41:20,204 INFO    MainThread:47009 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-26 02:41:20,210 INFO    MainThread:47009 [wandb_run.py:_restore():1466] restore
2021-04-26 02:41:22,399 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-04-26 02:41:22,523 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-04-26 02:41:22,663 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 4086
}

2021-04-26 02:41:22,802 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 4086
}

2021-04-26 02:41:22,905 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4086
  total_bytes: 4086
}

2021-04-26 02:41:23,019 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4086
  total_bytes: 4086
}

2021-04-26 02:41:23,130 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4086
  total_bytes: 4086
}

2021-04-26 02:41:23,237 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4086
  total_bytes: 4086
}

2021-04-26 02:41:23,340 INFO    MainThread:47009 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4086
  total_bytes: 4086
}

2021-04-26 02:41:24,529 INFO    MainThread:47009 [wandb_run.py:_show_files():1838] logging synced files
